src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/AuthorisedGetCaseViewOperation.java

```diff
@@ -74,7 +82,26 @@ private CaseView filterUpsertAccess(CaseType caseType, Set<String> userRoles, Ca
                                                                       userRoles,
                                                                       CAN_UPDATE)) {
             authorisedTriggers = new CaseViewTrigger[]{};
+
+            LOG.info("No authorised triggers for caseReference={} caseType={} version={} caseState={}, caseTypeACLs={}, caseStateACLs={} userRoles={}",
```

Comment:
filters can be removed because of 2 different conditions:
``` if (!getAccessControlService().canAccessCaseTypeWithCriteria(caseType,
                                                                     userRoles,
                                                                     CAN_UPDATE)
            || !getAccessControlService().canAccessCaseStateWithCriteria(caseView.getState().getId(),
                                                                      caseType,
                                                                      userRoles,
                                                                      CAN_UPDATE)) ```
With this log only do we know which of the two conditions caused the removal of the triggers?